docs/general/polkadot-v2.md

```diff
@@ -0,0 +1,234 @@
+different chains and using those chains’ collaborative potential to build inter-chain applications
+to solve real-world problems. Those applications will thus need to span across chains.
+
+**Increasingly fewer tasks will be handled by the relay-chain** that will focus efforts only on
+primary tasks: securing the network and providing secure message-passing capability.
+[System parachains](../learn/learn-system-chains.md) will be used to take over secondary relay-chain
+tasks such as staking, governance, etc.
+
+### XCM and Accords
+
+[XCM](../learn/learn-xcm-transport.md) is a format, a language of intention abstract over
+functionality common within chains. It creates an expressive language of what you intend to do or
+want to happen. There are no guarantees that chains faithfully interpret XCM messages; thus, XCM is
+not ideal if we are in a trustless environment.
+
+[XCMP](../learn/learn-xcm.md) is the transport layer for delivering XCM messages. It gives the
+transportation method and a secure route but not a framework for binding agreements.
+
+An **Accord** is an _opt-in_ treaty across many chains, where treaty logic cannot be changed or
+undermined by one or more of those chains, and Polkadot guarantees faithful execution of this logic.
+Accords will be specific to a particular function, and any chain that enters the accord will be held
+to it and will service that particular function. To lower the entry barrier, accords can be proposed
+permissionlessly, but because they are opt-in, the accord proposal will take effect until chains
+agree and sign up.
+
+Polkadot is the only ecosystem where accords can properly exist because it has an homogenous
+security layer that also provides a specific state transition function for each of its logic
+components. This allows patterns of cooperation between multiple logic components (i.e.,
+trans-applications) that would not be possible to achieve over bridges.
+
+Accords will be implemented using [SPREE technology](../learn/learn-spree.md).
+
+## Agile Coretime Allocation
+
+In Polkadot 1.0 coretime is a fixed two-year period on one specific core. Here we remove this
+limitation and generalize coretime usage to meet different application needs.
+
+### Split Coretime
+
+Owners of coretime can split or trade it. An application A1 can run on core C1 for a finite period
+and then another application A2 can run on that core, or application A1 can continue running on
+another core C2. Some applications might stop running for some time and resume later on.
+
+![core-usage-agile-rangeSplit](../assets/core-usage-agile-rangeSplit.png)
+
+### Strided Coretime
+
+Ranges can be strided (i.e., applications can take turns on a core) to share costs or decrease block
+production rate, for example.
+
+![core-usage-agile-rangeStrided](../assets/core-usage-agile-rangeStrided.png)
+
+### Combined Coretime
+
+An application's task can be assigned to multiple cores simultaneously. Some applications can have a
```

Comment:
We need to explain that "tasks" are essentially what we hitherto called "parachains"/"parathreads".

Reply:
removed task as application is here a broad term for parachains and parathreads